modules/standard/Sort.chpl

```diff
@@ -3302,41 +3435,51 @@ record DefaultComparator {
     else return 0;
   }
 
+  // TODO: remove and inline into `keyPart` when `useKeyPartStatus` is deprecated
+  @chpldoc.nodoc
+  inline proc chpl_keyPartInternal(elt: integral, i: int): (keyPartStatus, elt.type) {
+    var section = if i > 0 then keyPartStatus.pre else keyPartStatus.returned;
+    return (section, elt);
+  }
+
   /*
-   Default ``keyPart`` method for integral values.
-   See also `The .keyPart method`_.
+    Default ``keyPart`` method for integral values.
+    See also `The .keyPart method`_.
 
-   :arg x: the `int` or `uint` of any size to sort
-   :arg i: the part number requested
+    :arg elt: the `int` or `uint` of any size to sort
+    :arg i: the part number requested
 
-   :returns: ``(0, x)`` if ``i==0``, or ``(-1, x)`` otherwise
+    :returns: ``(keyPartStatus.pre, x)`` if ``i==0``, or ``(keyPartStatus.returned, x)`` otherwise
```

Comment:
I thought `0` was "returned" and `-1` was "pre", did you accidentally reverse these?

Reply:
D'oh!